packages/account/src/Components/poi/idv-document-submit/utils.js

```diff
@@ -34,6 +34,12 @@ export const isSequentialNumber = document_number => {
     return pattern_results.includes(true);
 };
 
+export const isIDVWhitelistDocumentNumber = (country, document_type, document_number) =>
+    (idv_test_document_whitelist.has(country) &&
+        idv_test_document_whitelist.get(country)[document_type] === document_number &&
+        getPlatformFromUrl().is_test_link) ||
+    getPlatformFromUrl().is_staging;
+
```

Comment:
Very nice! My only suggestion would be for now to add a comment on why this function is needed. Perhaps you can add a comment that says that this whitelists document numbers that QA needs for smileidentity sandbox testing?